content/docs/reference/config.mdx

```diff
@@ -55,4 +55,39 @@ export default defineConfig({
 ```
 
 In this example, the TinaCMS admin will be viewable at `<my-site-url>/admin/index.html`.
-For more information [check out the content modeling section](/docs/schema/).
+For more information [check out the content modeling section](/docs/schema/).\
+\
```

Comment:
**question**: Why are the backslashes required?

Reply:
Probably generated when i try to add a new line in between section